StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsAccountPickerPane.swift

```diff
@@ -0,0 +1,12 @@
+//
+//  FinancialConnectionsAccountPickerPane.swift
+//  StripeFinancialConnections
+//
+//  Created by Mat Schmid on 2024-09-12.
+//
+
+import Foundation
+
+struct FinancialConnectionsAccountPickerPane: Decodable {
+    let dataAccessNotice: String
```

Comment:
[it is non-null](https://git.corp.stripe.com/stripe-internal/pay-server/pull/896660/files#diff-3b2445d2897af15b93c7f5ad25f2e11741919016813ac19ff505cfa9b4555b6eR7) 👍